### DISCUSSION TITLE
Metamask not show the snap account!

### DISCUSSION THREAD

**Discussion by LaplaceMan** [MetaMask/snaps#1788] [Q&A] [answered]
I created the account using the snap provided by metamask, but it doesn't show up in my metamask
![image](https://github.com/MetaMask/snaps/assets/74650892/efa20153-4d0b-495e-bee5-955da57f8f05)

This is the desired effect：
![image](https://docs.metamask.io/assets/images/add-snap-account-dc3f67405f09d799d958d9829dd2189d.png)

**Comment by LaplaceMan** [comment]
@Montoya  I'd appreciate it if you could give me some clarification.

**Reply by Montoya** [reply]
I'll check with the team and let you know

**Reply by AlexJupiter** [reply, marked as answer]
@LaplaceMan what version of Flask are you using? If it's the one currently in the stores I'm afraid that the Snap you're trying to install is no longer compatible. You can test out a new build of Flask [here](https://github.com/MetaMask/metamask-extension/pull/20865#issuecomment-1730484127) that should be compatible with this Snap Simple Keyring.

**Reply by LaplaceMan** [reply]
Very thankful. Yes Currently the Metamsk I use is downloaded from the app store

**Reply by AlexJupiter** [reply]
Okay do let us know how you get on with that new build. Also please be aware that this Snap is just meant for development purposes, so do not transfer any real funds into this account.
